plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -933,7 +937,7 @@ private PlugNicAnswer execute(PlugNicCommand cmd) {
              */
```

Comment:
Interesting...this code does not appear to be commented out in the most recent version of the code on master.